src/esp/bindings/GfxReplayBindings.cpp

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+#include "esp/bindings/bindings.h"
+
+#include <Magnum/PythonBindings.h>
+#include <Magnum/SceneGraph/PythonBindings.h>
+
+#include "esp/gfx/replay/Player.h"
+#include "esp/gfx/replay/ReplayManager.h"
+
+namespace py = pybind11;
+using py::literals::operator""_a;
+
+namespace esp {
+namespace gfx {
+namespace replay {
+
+void initGfxReplayBindings(py::module& m) {
+  py::class_<Player, Player::ptr>(m, "Player")
+      .def("get_num_keyframes", &Player::getNumKeyframes,
```

Comment:
Pure getters and setters like "get_num_keyframes" and "get/set_keyframe_index" should be exposed with `def_property` and `def_property_readonly` as this better fits python

Reply:
Agreed!

Reply:
Just want to double-check here: in this case, set_keyframe_index is a very heavyweight operation--it might include creating render asset instances as necessary to restore the scene graph to the desired state.

Given that, do you still think a property is appropriate?

I want to discourage code like this:

```
player.keyframe_index = x
if some_condition:
  player.keyframe_index += 1
```  
I would prefer:
```
keyframe_index = x
if some_condition:
  keyframe_index += 1
player.set_keyframe_index(keyframe_index)
```

Reply:
Ah, good point.  Heavy weight properties can certainly be counter intuitive.

Reply:
I went ahead and merged this as-is but I realize I should change get_num_keyframes to be a read-only property. I'll make a separate PR soon.